installer/installer.go

```diff
@@ -45,6 +59,29 @@ func NewInstaller(l log.Logger) *Installer {
 	if err := installer.loadEventsFromDB(); err != nil {
 		shutdown.Fatalf("Error loading events from database: %s", err)
 	}
+
+	// Get current list of releases
+	resp, err := http.Get("https://releases.flynn.io/api/channels")
+	if err != nil {
+		l.Debug(fmt.Sprintf("Unable to fetch releases: %s", err))
+		defer resp.Body.Close()
```

Comment:
This should be outside of the `err != nil` block (if `err != nil` then `resp` could be `nil` and this will panic)